network/p2p/p2pnet/readSubscription.go

```diff
@@ -19,17 +19,17 @@ import (
 // ReadSubscriptionCallBackFunction the callback called when a new message is received on the read subscription
 type ReadSubscriptionCallBackFunction func(msg *message.Message, peerID peer.ID)
 
-// readSubscription reads the messages coming in on the subscription and calls the given callback until
+// ReadSubscription reads the messages coming in on the subscription and calls the given callback until
```

Comment:
Moved `ReadSubscription` to the `p2pnet/internal` package; hence it can remain exportable but is only accessible within the parent package: https://github.com/onflow/flow-go/pull/4664/commits/6eb91a99bcaf5a8739483a6de75b209549ea3630